packagingbuild/rockylinux8/Dockerfile

```diff
@@ -0,0 +1,98 @@
+FROM rockylinux:8
+
+# Download tools
+RUN yum -y install \
+    ca-certificates \
+    curl \
+    wget
+
+RUN yum -y install \git \
+    mercurial \
+    openssh \
+    subversion
```

Comment:
Just curious, are mercurial and subversion really still used here? 🙂

Reply:
@stevemuskiewicz Removed as weren't needed, and yes didn't need the \ above. Thanks for the review comments.